src/core/cgn.tf

```
@@ -122,3 +122,57 @@ module "cgn_cosmos_db" {
   account_name        = module.cosmos_cgn.name
 }
 
+
+## Blob storage due to legal backup
+#tfsec:ignore:azure-storage-default-action-deny
+// TODO: verify to use underscore or middlescore
+module "cgn-legalbackup-storage" {
```

```suggestion
module "cgn_legalbackup_storage" {
```
use `_` for terraform resource names